image.yaml

```diff
@@ -1,241 +1,28 @@
 name: "redhat-openjdk-18/openjdk18-openshift"
 description: "Source To Image (S2I) image for Red Hat OpenShift providing OpenJDK 1.8"
-version: "1.2"
+version: "2.0"
+schema_version: 1
+from: "jboss/openjdk18-rhel7:latest"
```

Comment:
For our base images we should define the stream number, just like for OCP images. In this case it should be still `1.0`.

Reply:
so, 1.2, since that's what it was?  maybe 1.3, next?

Reply:
We should base it on currently released version and bump version (to `1.1`) when it is necessary. Currently it's not necessary to release version `1.1` of that image because there is no feature in the development image that would be a requirements to build this image. We should bump image versions only if needed.

Reply:
You mean the image tag for the from image, not the version field, right?  If so, I think I've updated it correctly.